packages/algob/src/lib/tx.ts

```diff
@@ -292,10 +292,10 @@ export async function closeOutSSC (
   deployer: AlgobDeployer,
   sender: AccountSDK,
   payFlags: TxParams,
-  index: number): Promise<void> {
+  appId: number): Promise<void> {
   const params = await mkSuggestedParams(deployer.algodClient, payFlags);
 
-  const txn = tx.makeApplicationCloseOutTxn(sender, params, index);
+  const txn = tx.makeApplicationCloseOutTxn(sender, params, appId);
```

Comment:
please rename the function:
`makeApplicationCloseOutTxn` -> `mkAppCloseOutTxn`

Or.... how about using `mkASACloseOut`? (Txn is probably not needed, because we use `tx` namespace`

Reply:
how do we change this name ?
this function is called using `makeApplicationCloseOutTxn` name in `algosdk`.
we are using tx here as `import tx from "algosdk";`

I think it will call to nothing because there won't be no function with `mkAppCloseOutTxn` name in `algosdk`

Reply:
ah, sorry. I thought it's our method. Then please ignore my comment above.